test/e2e/traceflow_test.go

```diff
@@ -1088,11 +1108,26 @@ func testTraceflowIntraNode(t *testing.T, data *TestData) {
 
 // testTraceflowInterNode verifies if traceflow can trace inter nodes traffic with some NetworkPolicies set.
 func testTraceflowInterNode(t *testing.T, data *TestData) {
-	node1 := nodeName(0)
-	node2 := nodeName(1)
+	encapMode, err := data.GetEncapMode()
+	if err != nil {
+		t.Fatalf("Failed to retrieve encap mode: %v", err)
+	}
+	if encapMode != config.TrafficEncapModeNoEncap {
+		// https://github.com/antrea-io/antrea/issues/897
+		skipIfProviderIs(t, "kind", "Skipping inter-Node Traceflow test for Kind because of #897")
+	}
 
-	node1Pods, _, node1CleanupFn := createTestBusyboxPods(t, data, 1, testNamespace, node1)
-	node2Pods, node2IPs, node2CleanupFn := createTestBusyboxPods(t, data, 2, testNamespace, node2)
+	nodeIdx0 := 0
+	nodeIdx1 := 1
+	if len(clusterInfo.windowsNodes) != 0 {
```

Comment:
Should check the number of Windows Nodes is not less than 2?

Reply:
Fixed. Thanks.